README.md

```diff
@@ -1 +1,61 @@
-# chrony
\ No newline at end of file
+[![Build Status](https://travis-ci.org/oasis-roles/chrony.svg?branch=master)](https://travis-ci.org/oasis-roles/chrony)
+
+ROLE NAME
+===========
+
+Basic description for chrony
+
+Requirements
+------------
+
+Ansible 2.4 or higher
+
+Red Hat Enterprise Linux 7 or equivalent
+
+Valid Red Hat Subscriptions
+
+Role Variables
+--------------
+
+Currently the following variables are supported:
+
+### General
+
+* `chrony_package` - The package name to install on remote hosts (set to `chrony` by default)
+* `chrony_become_user` - The user to sign into when running commands on remote hosts (set to `root` by default)
+* `chrony_conf_template_path` - Path to a jinja2 template for generating the chrony configuration file (set to `chrony.conf` by default)
```

Comment:
I'm not so sure we need to provide the user a configurable path to the template. The template is something the role should just own and maintain, with its values fillable by configuration options.